mx.sulong/mx_sulong.py

```diff
@@ -419,13 +426,30 @@ def runTypeTestCases(args=None):
     vmArgs, _ = truffle_extract_VM_args(args)
     return unittest(getCommonUnitTestOptions() + vmArgs + ['com.oracle.truffle.llvm.types.floating.test'])
 
-def getCommonOptions():
-    return ['-Dgraal.TruffleCompilationExceptionsArePrinted=true', '-Dgraal.ExitVMOnException=true', '-Dsulong.IntrinsifyCFunctions=false', getSearchPathOption()]
+default_lib_args = ['-lgmp', '-lgfortran']
+
+def getCommonOptions(lib_args=default_lib_args):
```

Comment:
I think it already does, doesn't it? It calls `getSearchPathOption()` with no argument, so it gets `default_lib_args`, which is `['-lgmp', '-lgfortran']`.